cowbird/handlers/impl/filesystem.py

```diff
@@ -40,7 +40,7 @@ def _get_user_workspace_dir(self, user_name):
     def user_created(self, user_name):
         user_workspace_dir = self._get_user_workspace_dir(user_name)
         try:
-            os.mkdir(user_workspace_dir)
+            os.mkdir(user_workspace_dir, 0o755)
```

Comment:
Consider a separate `chmod` in the `FileExistsError` case if it did not have the right permission.